source/documentation/install-guide/chap-Enterprise_Linux_Hosts.html.md

```diff
@@ -8,24 +8,35 @@ title: Enterprise Linux Hosts
 
 An Enterprise Linux host (such as CentOS or RHEL), also known as an EL-based hypervisor is based on a standard basic installation of an Enterprise Linux operating system on a physical server.
 
-Ensure that you have required repositories for your distribution.
-On CentOS the Base, Optional and Extras repositories are already enabled by default and must be enabled.
-On Red Hat Enterprise Linux you'll need a valid subscription and the following repositories enabled:
+The host must meet the minimum Host requirements out lined in [Chapter 2: System Requirements](../chap-System_Requirements).
```

Comment:
I think that the section about needed repositories and yum install of ovirt-release42 rpm is still needed here

Reply:
Can upstream oVirt users get to them? Can you help with the appropriate text?

Reply:
@sandrobonazzola Ping.